distribution/lib/Standard/Base/0.0.0-dev/src/Data/Index_Sub_Range.enso

```diff
@@ -55,6 +56,17 @@ type Index_Sub_Range
          input, an error is raised.
     Every (step:Integer) (first:Integer=0)
 
+    ## PRIVATE
+       Convert to a display representation of this Index_Sub_Range.
+    to_display_text : Text
+    to_display_text self = case self of
+        Index_Sub_Range.First count -> "First " + count.to_display_text
+        Index_Sub_Range.Last count -> "Last " + count.to_display_text
+        Index_Sub_Range.While _ -> "While <predicate>"
```

Comment:
Why not?

```suggestion
        Index_Sub_Range.While f -> "While "+f.to_display_text
```

I see that currently `to_display_text` of a `Function` returns `"Function"`. That is not helpful indeed. But if we changed it to return the same thing as `to_text`, then it could actually be useful - function's `to_text` shows the _name_ and _source location_ of the function. It can be of some help to identify the predicate.

Not a big deal though, just a suggestion of an improvement I thought of.